prod/westeurope/internal/api/functions_assets_r3/function_assets/terragrunt.hcl

```diff
@@ -0,0 +1,101 @@
+dependency "cosmosdb_account" {
+  config_path = "../../cosmosdb/account"
+}
+
+dependency "cosmosdb_database" {
+  config_path = "../../cosmosdb/database"
+}
+
+dependency "storage_account" {
+  config_path = "../../storage/account"
+}
+
+dependency "storage_account_assets" {
+  config_path = "../../../../common/cdn/storage_account_assets"
+}
+
+dependency "storage_account_logs" {
+  config_path = "../../../../operations/storage_account_logs/account"
+}
+
+# Internal
+dependency "resource_group" {
+  config_path = "../../../resource_group"
+}
+
+# Common
+dependency "virtual_network" {
+  config_path = "../../../../common/virtual_network"
+}
+
+dependency "application_insights" {
+  config_path = "../../../../common/application_insights"
+}
+
+dependency "key_vault" {
+  config_path = "../../../../common/key_vault"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_function_app?ref=v2.1.0"
+}
+
+inputs = {
+  name                = "assets"
+  resource_group_name = dependency.resource_group.outputs.resource_name
+
+  resources_prefix = {
+    function_app     = "fn3"
+    app_service_plan = "fn3"
+    storage_account  = "fn3"
+  }
+
+  app_service_plan_info = {
+    kind     = "elastic"
+    sku_tier = "ElasticPremium"
+    sku_size = "EP1"
```

Comment:
```suggestion
    sku_size = "S1"
```

Reply:
In this way we are going to save some money.

Reply:
Done in [d6d9a29](https://github.com/pagopa/io-infrastructure-live-new/pull/261/commits/d6d9a29392867b01c202f2dbb7035c8a3c9d02f4)